### ISSUE TITLE
Install against conda-forge tempo2 is broken

### ISSUE THREAD

**Issue by vallis** [conda-forge/libstempo-feedstock#17]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Loading a pulsar results in a `UnicodeDecodeError`. That's not the case when installing with pip or setup.py.

```
>>> import libstempo
AWARNING: AstropyDeprecationWarning: The private astropy._erfa module has been made into its own package, pyerfa, which is a dependency of astropy and can be imported directly using "import erfa" [astropy._erfa]

>>> out = libstempo.tempopulsar("par/B1937+21_NANOGrav_12yv3.gls.par", "tim/B1937+21_NANOGrav_12yv3.tim")
[tempo2Util.C:396] Warning: [MISC1] Unknown parameter in par file:  DMX
[tempo2Util.C:401] Warning: [DUP1] duplicated warnings have been suppressed.

WARNING: Using tempo RNAMP parameter: setting TNRedC to 100!
[tempo2Util.C:396] Warning: [PAR2] Have not set a DM epoch. The period epoch will be used instead. par/B1937+21_NANOGrav_12yv3.gls.par
[tempo2Util.C:396] Warning: [TIM1] Please place MODE flags in the parameter file 
[observatory.C:271] Warning: Assuming site '1' means 'gbt'
[observatory.C:271] Warning: Assuming site '3' means 'ao'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "libstempo/libstempo.pyx", line 728, in libstempo.libstempo.tempopulsar.__cinit__
  File "libstempo/libstempo.pyx", line 875, in libstempo.libstempo.tempopulsar._readflags
  File "libstempo/libstempo.pyx", line 23, in libstempo.libstempo.lambda1
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x91 in position 2: invalid start byte
```

### Installed packages

```shell
# packages in environment at /ocean/projects/phy210013p/vallis/tempo2:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                      1_llvm    conda-forge
astropy                   5.0.1            py38h6c62de6_0    conda-forge
brotli                    1.0.9                h7f98852_6    conda-forge
brotli-bin                1.0.9                h7f98852_6    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
certifi                   2021.10.8        py38h578d9bd_1    conda-forge
cfitsio                   4.0.0                h9a35b8e_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cython                    0.29.28          py38h709712a_0    conda-forge
dbus                      1.13.18              hb2f20db_0  
ephem                     4.1.3                    pypi_0    pypi
expat                     2.4.6                h27087fc_0    conda-forge
fftw                      3.3.10          nompi_h77c792f_102    conda-forge
fontconfig                2.13.96              ha180cfb_0    conda-forge
fonttools                 4.29.1           py38h497a2fe_0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
fribidi                   1.0.10               h36c2ea0_0    conda-forge
giflib                    5.2.1                h36c2ea0_2    conda-forge
glib                      2.69.1               h4ff587b_1  
gsl                       2.7                  he838d99_0    conda-forge
gst-plugins-base          1.14.0               hbbd80ab_1  
gstreamer                 1.14.0               h28cd5cc_2  
icu                       58.2              hf484d3e_1000    conda-forge
importlib-metadata        4.11.2           py38h578d9bd_0    conda-forge
jbig                      2.1               h7f98852_2003    conda-forge
jpeg                      9e                   h7f98852_0    conda-forge
keyutils                  1.6.1                h166bdaf_0    conda-forge
kiwisolver                1.3.2            py38h1fd1430_1    conda-forge
krb5                      1.19.2               h3790be6_4    conda-forge
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.35.1               h7274673_9  
lerc                      3.0                  h9c3ff4c_0    conda-forge
libblas                   3.9.0           13_linux64_openblas    conda-forge
libbrotlicommon           1.0.9                h7f98852_6    conda-forge
libbrotlidec              1.0.9                h7f98852_6    conda-forge
libbrotlienc              1.0.9                h7f98852_6    conda-forge
libcblas                  3.9.0           13_linux64_openblas    conda-forge
libcurl                   7.81.0               h2574ce0_0    conda-forge
libdeflate                1.10                 h7f98852_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0              h1d223b6_13    conda-forge
libgfortran-ng            11.2.0              h69a702a_13    conda-forge
libgfortran5              11.2.0              h5c6108e_13    conda-forge
libimagequant             2.17.0               h7f98852_1    conda-forge
liblapack                 3.9.0           13_linux64_openblas    conda-forge
libnghttp2                1.47.0               h727a467_0    conda-forge
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-ng              11.2.0              he4da1e4_13    conda-forge
libtiff                   4.3.0                h542a066_3    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libwebp                   1.2.2                h3452ae3_0    conda-forge
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.13              h7f98852_1004    conda-forge
libxml2                   2.9.12               h03d6c58_0  
libzlib                   1.2.11            h36c2ea0_1013    conda-forge
llvm-openmp               13.0.1               he0ac6c6_1    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
matplotlib                3.5.1            py38h578d9bd_0    conda-forge
matplotlib-base           3.5.1            py38hf4fb855_0    conda-forge
mkl                       2022.0.1           h8d4b97c_803    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
ncurses                   6.3                  h7f8727e_2  
numpy                     1.22.3           py38h05e7239_0    conda-forge
openjpeg                  2.4.0                hb52868f_1    conda-forge
openssl                   1.1.1l               h7f98852_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.4.1            py38h43a58ef_0    conda-forge
pcre                      8.45                 h9c3ff4c_0    conda-forge
pgplot                    5.2.2             h68245ad_1008    conda-forge
pillow                    9.0.1            py38he2f12e7_1    conda-forge
pip                       21.2.4           py38h06a4308_0  
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pyerfa                    2.0.0.1          py38h6c62de6_1    conda-forge
pyparsing                 3.0.7              pyhd8ed1ab_0    conda-forge
pyqt                      5.9.2            py38h05f1152_4  
python                    3.8.12               h12debd9_0  
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pyyaml                    6.0              py38h497a2fe_3    conda-forge
qt                        5.9.7                h5867ecd_1  
readline                  8.1.2                h7f8727e_1  
scipy                     1.8.0            py38h56a6a73_1    conda-forge
setuptools                58.0.4           py38h06a4308_0  
sip                       4.19.13          py38h295c915_0  
six                       1.16.0             pyh6c4a22f_0    conda-forge
sqlite                    3.37.2               hc218d9a_0  
tbb                       2021.5.0             h4bd325d_0    conda-forge
tempo2                    2022.01.1            h4634e15_0    conda-forge
tk                        8.6.11               h1ccaba5_0  
tornado                   6.1              py38h497a2fe_2    conda-forge
unicodedata2              14.0.0           py38h497a2fe_0    conda-forge
wheel                     0.37.1             pyhd3eb1b0_0  
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libx11               1.7.2                h7f98852_0    conda-forge
xorg-libxau               1.0.9                h7f98852_0    conda-forge
xorg-libxdmcp             1.1.3                h7f98852_0    conda-forge
xorg-xproto               7.0.31            h7f98852_1007    conda-forge
xz                        5.2.5                h7b6447c_0  
yaml                      0.2.5                h7f98852_2    conda-forge
zipp                      3.7.0              pyhd8ed1ab_1    conda-forge
zlib                      1.2.11            h36c2ea0_1013    conda-forge
zstd                      1.5.2                ha95c52a_0    conda-forge
```

### Environment info

```shell
active environment : /ocean/projects/phy210013p/vallis/tempo2
    active env location : /ocean/projects/phy210013p/vallis/tempo2
            shell level : 2
       user config file : /jet/home/vallis/.condarc
 populated config files : 
          conda version : 4.10.3
    conda-build version : 3.20.5
         python version : 3.8.5.final.0
       virtual packages : __linux=4.18.0=0
                          __glibc=2.28=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /opt/packages/anaconda3  (read only)
      conda av data dir : /opt/packages/anaconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /jet/home/vallis/.conda/pkgs
       envs directories : /jet/home/vallis/.conda/envs
                          /opt/packages/anaconda3/envs
               platform : linux-64
             user-agent : conda/4.10.3 requests/2.24.0 CPython/3.8.5 Linux/4.18.0-193.28.1.el8_2.x86_64 centos/8.2.2004 glibc/2.28
                UID:GID : 83839:24880
             netrc file : None
           offline mode : False
```

**Comment by mattpitkin** [comment]
I noticed issues with the latest version of tempo2 as well. I'll take a look and see if I can fix things, otherwise I might just have to pin the conda-forge build to the previous version.

**Comment by vallis** [comment]
Presumably a mismatch between the tempo2 and libstempo conda dependencies... since libstempo builds fine manually against the latest conda tempo2.

**Comment by mattpitkin** [comment]
I think I've fixed this now in https://github.com/conda-forge/libstempo-feedstock/pull/18. This also now runs the libstempo test suite when building the conda forge release. There are a couple of patches that I had to manually add that I will make a PR for within libstempo.

I just need to wait a bit longer for the new build to propagate through and double check that too.

**Comment by mattpitkin** [comment]
@vallis it looks like the updated version is available now. Could you try uninstalling libstempo from your conda environment, reinstalling it and trying your test again? It seems to work ok for me.

**Comment by vallis** [comment]
Hmm... having trouble commenting.

Seems to work now. With Python 3.8 on Linux, in a fresh conda environment, with the sequence

**Comment by mattpitkin** [comment]
Fixed by #18
